tests/integration/targets/setup_openldap/tasks/main.yml

```
@@ -0,0 +1,63 @@
+---
+- name: Setup OpenLDAP on Debian or Ubuntu
+  block:
+    - name: Include OS-specific variables
+      include_vars: '{{ ansible_os_family }}.yml'
+
+    - name: Install OpenLDAP server and tools
+      become: True
+      package:
+        name: '{{ item }}'
+      loop: '{{ openldap_packages_name }}'
+
+    - name: Install python-ldap (Python 3)
+      become: True
+      package:
+        name: '{{ python_ldap_package_name_python3 }}'
+      when: ansible_python_version is version('3.0', '>=')
+
+    - name: Install python-ldap (Python 2)
+      become: True
+      package:
+        name: '{{ python_ldap_package_name }}'
+      when: ansible_python_version is version('3.0', '<')
+
+    - name: Make sure OpenLDAP service is stopped
+      become: True
+      shell: 'cat /var/run/slapd/slapd.pid | xargs kill -9 '
+
+    - name: Debconf
+      shell: "echo \"slapd {{ item.question }} {{ item.vtype }} {{ item.value }}\" >> /root/debconf-slapd.conf"
```

```suggestion
      shell: 'echo "slapd {{ item.question }} {{ item.vtype }} {{ item.value }}" >> /root/debconf-slapd.conf'
```
Then there's no need to escape.